devices/YarpRobotLoggerDevice/app/robots/iCubGazeboV3/yarp-robot-logger.xml

```diff
@@ -22,6 +23,15 @@ GNU Lesser General Public License v2.1 or any later version. -->
 
   </group>
 
+  <group name="RobotCameraBridge">
+      <param name="stream_cameras">false</param>
+      <group name="Cameras">
+        <param name="rgb_cameras_list">("jabra", "logitech")</param>
+        <param name="rgb_image_width">(1280, 1280)</param>
+        <param name="rgb_image_height">(720, 720)</param>
+      </group>
+  </group>
```

Comment:
What happens if this group is available in config but the cameras are commented in the `attach` phase, as done below?